### PR TITLE
polish views of League

### PR DESCRIPTION
**Pull request by edudepetris**
There are a few Bootstrap classes that I had hard-code as ".nav.nav-tabs, tab-content..." :warning: but this is my first encounter with Less. :alien:

### CODE REVIEW

File: app/views/leagues/_edit_teams_table.html.haml [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-            .text-left
-              = action == "affiliate" ? I18n.t("leagues.edit.action_affiliate") : I18n.t("leagues.edit.action_delete")
-        - collection.each do |team|
-          = render "row_edit_team_table", team: team, action: action
+%div{ id: "#{action}-table"}
+  %table.teams-league
+    %thead
+      %th
+        .text-left
+          = I18n.t("leagues.edit.team_header_row")
```
Comment (fbrusatti) [inline app/views/leagues/_edit_teams_table.html.haml]: I think that in views there is not need to use `I18n` you have to use just `t`

File: app/views/leagues/_edit_teams_table.html.haml [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
+      %th
+        .text-left
+          = I18n.t("leagues.edit.owner_header_row")
```
Comment (fbrusatti) [inline app/views/leagues/_edit_teams_table.html.haml]: the same here

File: app/views/leagues/_edit_teams_table.html.haml [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
-          %th
-            .text-left
-              = action == "affiliate" ? I18n.t("leagues.edit.action_affiliate") : I18n.t("leagues.edit.action_delete")
-        - collection.each do |team|
-          = render "row_edit_team_table", team: team, action: action
+%div{ id: "#{action}-table"}
+  %table.teams-league
+    %thead
+      %th
+        .text-left
```
Comment (fbrusatti) [inline app/views/leagues/_edit_teams_table.html.haml]: I think that `.text-left` is not semantic enough. You should have to use more descriptive names.

For example 

```
%th
   .team-header
     = t('leagues.edit.team_header_row')
```

and https://github.com/fbrusatti/fulbo/pull/63/files#L0R164 should be

```
.team-header, .owner-header, .action-header {
  .text-left;
}
```

File: app/views/leagues/_edit_teams_table.html.haml [hunk 4]
```diff
@@ -1,16 +1,14 @@
...
+          = I18n.t("leagues.edit.team_header_row")
+      %th
+        .text-left
```
Comment (fbrusatti) [inline app/views/leagues/_edit_teams_table.html.haml]: ``` .owner-header```
```

File: app/views/leagues/_edit_teams_table.html.haml [hunk 5]
```diff
@@ -1,16 +1,14 @@
...
+          = I18n.t("leagues.edit.owner_header_row")
+      %th
+        .text-left
```
Comment (fbrusatti) [inline app/views/leagues/_edit_teams_table.html.haml]: ``` .action-header```
```

**Comment by edudepetris** [comment]
@fbrusatti  :eyes:

**Comment by fbrusatti** [comment]
:goat:

**Pull request merged**